packages/page-accounts/test/pages/accountsPage.tsx

```diff
@@ -96,7 +96,7 @@ export class AccountsPage {
             <MemoryRouter>
               <ThemeProvider theme={lightTheme}>
                 <ApiContext.Provider value={mockApi}>
-                  <Overview/>
+                  <Overview onStatusChange={() => undefined}/>
```

Comment:
useCallback.

Reply:
This is a class we use for tests, we can't use hooks in class components. I've extracted this to mock function or should I handle this somehow different?